next.config.js

```diff
@@ -0,0 +1,17 @@
+const withCSS = require("@zeit/next-css");
+const withSass = require("@zeit/next-sass");
+const withFonts = require("next-fonts");
+const withTM = require("next-transpile-modules")(["react-rainbow-components"]);
+
+module.exports = withTM(
+  withFonts(
+    withCSS(
+      withSass({
+        enableSvg: true,
+        webpack: (config, options) => {
+          return config;
+        },
+      })
+    )
+  )
+);
```

Comment:
there is a way we can do this without config? try to use material UI with next js to see if there is any config necessary and if it is possible what we want is that the consumer use react-rainbow-components with next out of the box, without any config